Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs

```diff
@@ -15,12 +15,15 @@ public class receiveSignalLockedDoor : receiveSignal {
 
     public override void activate() {
         animator.SetBool("open", true);
-		doorToNextLevel.setOpen(true);
-        
+        if (doorToNextLevel != null) {
```

Comment:
This is bad code that I did late at night lol. Basically I reused the locked door that was in 1b, and that door would set a bool that would allow the player to open the door to advance to the next level. As the door in 3a isn't connected to the exit door, it doesn't have a doorToNextLevel.

I would say merge this for now but create a task to refactor it later